corehq/apps/userreports/README.rst

```diff
@@ -1837,6 +1837,51 @@ Rounds decimal and floating point numbers to two decimal places.
        "custom_type": "short_decimal_display"
    }
 
+
+Prefix a string
+~~~~~~~~~~~~~~~
+
+Prefixes the column value with another string
+
+.. code:: json
+	{
+        "type": "prefix_string",
+        "prefix": "hello "
+    }
```

Comment:
Do you see this being useful besides in link generation? Any reason you didn't so something more akin to string formatting which might be more general purpose?